app/views/layouts/application.html.haml

```diff
@@ -14,8 +14,8 @@
       = yield
     = render partial: 'layouts/footer'
 
-:javascript
-  if(#{(current_user && current_user.location.is_geocodable) || false}) {
-    track_location();
-  }
+  :javascript
+    if(#{(current_user && current_user.location.is_geocodable) || false}) {
+      track_location();
+    }
```

Comment:
Consider removing this JS for users it doesn't apply to on the server-side. This will reduce the data and work the browser has to receive and process (admittedly it'll be a very very minor reduction). You may also find the code more succinct and understandable:

```haml
 - if current_user && current_user.location.is_geocodable
  :javascript
    track_location();
```

Other thoughts:
- Conventionally JS naming is camelCase. Compare `trackLocation()` to `track_location()`
- Consider introducing a `geocodable?` (or other name) helper method on ApplicationHelper that will reduce the logic to:

```haml
  - if geocodable?(current_user)
```
- In Ruby, its more conventional to name boolean methods with a `?` suffix rather than an `is_` prefix. For example `location.geocodable?` reads a little more Ruby-ish/conventional than `location.is_geocodable`.